xla/tests/llvm_compiler_test.cc

```diff
@@ -45,8 +49,13 @@ constexpr char kDummyLayout[] = "e";
 class GpuDummyCompiler : public GpuCompiler {
  public:
   GpuDummyCompiler()
-      : GpuCompiler(se::cuda::kCudaPlatformId, kDummyTriple, kDummyLayout) {}
-
+      : GpuCompiler(
+#if GOOGLE_CUDA         
```

Comment:
I think we would need local_defines for the test?

Reply:
Thanks for the feedback. Do you mean `local_defines` in bazel BUILD?

Reply:
Yes. I think we should not rely on having GOOGLE_CUDA and TENSORFLOW_USE_ROCM be defined elsewhere.

Reply:
Done